skactiveml/pool/regression/_query_by_committee.py

```diff
@@ -0,0 +1,181 @@
+from copy import deepcopy
+
+import numpy as np
+from sklearn import clone
+from sklearn.utils.validation import _is_arraylike
+
+from skactiveml.base import (
+    SingleAnnotatorPoolQueryStrategy,
+    SkactivemlRegressor,
+)
+from skactiveml.pool.regression.utils._model_fitting import bootstrap_estimators
+from skactiveml.utils import (
+    simple_batch,
+    check_type,
+    check_scalar,
+    MISSING_LABEL,
+)
+
+
+class QueryByCommittee(SingleAnnotatorPoolQueryStrategy):
+    """Regression based Query-by-Committee.
+
+    This class implements an Regression adaption of Query by Committee. It
+    tries to estimate the model variance by a Committee of estimators.
+
+    Parameters
+    ----------
+    k_bootstraps: int, optional (default=3)
+        The number of bootstraps used to estimate the true model.
+    n_train: int or float, optional (default=0.5)
+        The size of a bootstrap compared to the training data.
+    missing_label : scalar or string or np.nan or None, default=np.nan
+        Value to represent a missing label.
+    random_state: numeric | np.random.RandomState, optional
+        Random state for candidate selection.
+
+    References
+    ----------
+    [1] Burbidge, Robert and Rowland, Jem J and King, Ross D. Active learning
+        for regression based on query by committee. International conference on
+        intelligent data engineering and automated learning, pages 209--218,
+        2007.
+
+    """
+
+    def __init__(
+        self,
+        k_bootstraps=3,
+        n_train=0.5,
+        missing_label=MISSING_LABEL,
+        random_state=None,
+    ):
+        super().__init__(random_state=random_state, missing_label=missing_label)
+        self.k_bootstraps = k_bootstraps
+        self.n_train = n_train
+
+    def query(
+        self,
+        X,
+        y,
+        ensemble,
+        fit_ensemble=True,
+        sample_weight=None,
+        candidates=None,
+        batch_size=1,
+        return_utilities=False,
+    ):
+        """Determines for which candidate samples labels are to be queried.
+
+        Parameters
+        ----------
+        X : array-like of shape (n_samples, n_features)
+            Training data set, usually complete, i.e. including the labeled and
+            unlabeled samples.
+        y : array-like of shape (n_samples)
+            Labels of the training data set (possibly including unlabeled ones
+            indicated by `self.missing_label`).
+        ensemble: {SkactivemlRegressor, array-like}
+            Regressor to predict the data.
+        fit_ensemble : bool, optional (default=True)
+            Defines whether the classifier should be fitted on `X`, `y`, and
+            `sample_weight`.
```

Comment:
I assume that the description is not correct or I do not understand why we are talking about a `classifier`